### DISCUSSION TITLE
I cant seem to have auto resolution

### DISCUSSION THREAD

**Discussion by Jamesluther** [JaKooLit/Fedora-Hyprland#32] [General]
Hi everyone! after installed hyprland I tried going into to monitor config file to change the resolution but there is nothing there. Do you have any Ideas. this is the command.

nano .config/hypr/configs/Monitors.conf/

**Comment by JaKooLit** [comment]
should be 

~/.config/hypr/UserConfigs/Monitors.conf

https://github.com/JaKooLit/Hyprland-Dots/wiki

I have created a wiki for the dotfiles

**Reply by Jamesluther** [reply]
Great thank you. That fixed it. One last question. Everything is very pixelated is there a way to fix that?

**Reply by JaKooLit** [reply]
> Great thank you. That fixed it. One last question. Everything is very pixelated is there a way to fix that?

hopefully you found solution here

![swappy-20240524-132032](https://github.com/JaKooLit/Fedora-Hyprland/assets/85185940/27da58db-40d1-4860-a59e-0a1229cdbebd)
